roles/streambackup/templates/streambackup.sh.j2

```diff
@@ -7,12 +7,12 @@ while :
 do
     timestamp=`date +%Y-%m-%d-%H%M%S`
     backupName="{{ channel_name }}-backup-$timestamp.flv"
-    streamStatus=`curl -s -H "Streamdelay-API-Key:$(jq .apiKey {{ streamdelay_config_path }} | sed 's/"//g')" http://127.0.0.1:8404/status | jq .state.stream`
+    streamStatus=`curl -s -H "Streamdelay-API-Key:$(jq .apiKey {{ streamdelay_config_path }} | sed 's/"//g')" http://{{ streamdelay_hostname }}:{{ streamdelay_port }}/status | jq .state.stream`
     if [[ $streamStatus == "\"stopped\"" ]]; then
         sleep 0.5
     else
         timeout $timeoutTime gst-launch-1.0 -v -e \
-            srtsrc uri=srt://{{ streamdelay_host }}:{{ streamdelay_port }}?mode=caller ! queue ! tsparse set-timestamps=true ! tsdemux name=demux \
+            srtsrc uri=srt://{{ streamdelay_in_uri }}?mode=caller ! queue ! tsparse set-timestamps=true ! tsdemux name=demux \
```

Comment:
Please remove `?mode=caller` here since it's included in `streamdelay_in_uri`

Reply:
Got it.